CONTRIBUTING.md

```diff
@@ -0,0 +1,162 @@
+
+[Code of Conduct](#code-of-conduct)
+
+[Help! I don't want to read this whole thing, I just have one question. :mag_right:](#help!-i-dont-want-to-read-this-whole-thing-i-just-have-a-question)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Pull Requests](#pull-requests)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [JavaScript Styleguide](#javascript-styleguide)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Lisk-JS Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [info@lisk.io](mailto:info@lisk.io).
+
+## Project License
+
+Every repository within LiskHQ comes with a LICENSE file. Please read it carefully before commiting your code to one of the repositories.
+
+
+## Help! I don't want to read this whole thing, I just have a question. :mag_right:
+
+Lisk is an open-source decentralized project, there are many ways and platforms to get help. These are some of them
+
+* [Discuss and Ask on Reddit](https://www.reddit.com/r/Lisk/)
+* [Lisk FAQ](https://docs.lisk.io/docs/faq)
+
+If you prefer to chat with LiskHQ and other developers directly::
+
+* [Join the LiskHQ Gitter](https://gitter.im/LiskHQ/lisk)
+* Even though Gitter is a chat service, sometimes it takes several hours for community members to respond &mdash; please be patient!
+
+## How Can I Contribute?
+
+LiskHQ uses GitHub as its sole source of truth. Everything happens here. Lightcurve employees who contribute to Lisk are expected to do so in the same way as everyone else. In other words, this document applies equally to all contributors.
+
+### `master` is unsafe
+
+We will do our best to keep `master` in good shape, with tests passing at all times. But in order to move fast, we will make API changes that your application might not be compatible with. We will do our best to communicate these changes and always version appropriately so you can lock into a specific version if need be.
+
+### Pull Requests
+
+In case you've never submitted a pull request (PR) via GitHub before, please read [this short tutorial](https://help.github.com/articles/creating-a-pull-request). If you've submitted a PR before, there should be nothing surprising about our procedures for Lisk.
+
+*Before* submitting a pull request, please make sure the following is done:
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests!
+3. Ensure the test suite passes (`npm test`).
+4. Submit a pull request via GitHub.
+5. Check that Jenkins CI tests pass (pull request turns green).
+New developers probably need to wait for a trusted team member to start Jenkins CI on a Pull Request
+
+* Include issue numbers in the PR title, at the end with: ```Description - Closes #IssueNumber```
```

Comment:
Why is there a mix of ordered and unordered lists? Does the second list need some introduction or is it really part of the first list?